app/views/content/subjects/physics.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Teach physics
```

Comment:
Our maths page has a title (different to h1) of `Become a maths teacher` - which works better for SEO?

Reply:
Advice from @EllieNodder is that 'Teach physics' has a slightly higher search volume, but only just!

Reply:
Worth relooking at the maths title when you go back to that page again @philspray ?

Reply:
Yep absolutely!